web-server/routes/profile.test.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+describe('/profile', function(){
+  it('should render the current users profile', function(done){
+    webServer()
```

Comment:
please use the `return a promise` method of doing async mocha tests. instead of using `done` see the test for `/web-server/index.test.js` for an example